File: timeish-app/src/app/models/employee.ts
export class Employee {
    id: number;
    firstName: string;
    lastName: string;
    fullName: string;
    email: string;
    hourlyPay: number;
    constructor(employeeInstance) {
        Object.assign(this, employeeInstance);
        this.fullName = this.firstName.concat(' ', this.lastName);
    }
}

File: src/Tymish.Core/Entities/Employee.cs
using System;
using System.Runtime.Serialization;

namespace Tymish.Core.Entities
{
    public class Employee : BaseEntity
    {
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Email { get; set; }
        public Decimal HourlyPay { get; set; }
        // Do not serialize to Front End
        [IgnoreDataMember] protected string HashedPassword { get; set; }
    }
}